recipes-qt/qt5/qt5-git.inc

```diff
@@ -3,7 +3,7 @@
 
 QT_MODULE ?= "${BPN}"
 QT_MODULE_BRANCH ?= "5.11"
-QT_MODULE_BRANCH_PARAM ?= "branch=${QT_MODULE_BRANCH}"
+QT_MODULE_BRANCH_PARAM ?= "branch=${QT_MODULE_BRANCH};nobranch=1"
```

Comment:
No, we talked about this on ML already.

Reply:
As far as I remember, there was no reason why this could not be included?

Reply:
Is this still about the claim of AUTOREV build breaking? Like I replied earlier, nobranch does not affect it and it works fine.

Reply:
It does affect it, because with incorrect branch parameter AUTOREV will track wrong branch.

Reply:
I could change QT_MODULE_BRANCH to 5.11.2, but after next week both regular builds and AUTOREV builds would be broken once the branches get removed.
AUTOREV is already tracking "wrong" branch, since 5.11 is already 5.11.3 material. nobranch=1 doesn't change that in any way.